packages/components/src/components/option/option.ts

```diff
@@ -157,7 +157,7 @@ export default class SdOption extends SolidElement {
               id="control"
               part="control ${this.selected ? ' control--checked' : 'control--unchecked'}"
               class=${cx(
-                'relative flex flex-initial items-center justify-center border rounded-sm h-4 w-4 mr-2',
+                'relative flex flex-shrink-0 items-center justify-center border rounded-sm h-4 w-4 mr-2',
```

Comment:
I think it needs to be fixed in the checkbox itself... testing it there, still shows the bug:


Uploading Screen Recording 2024-09-17 at 18.50.49.mov…

Reply:
@karlbaumhauer From what I see, you are manually adding a width element to the checkbox, the size changing there should be expected. I think a better test would be to change the width of the window with your cursor, or adding even more text to the slot. In both cases, the checkbox alone retains it's size. 
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/3cdab728-4312-4477-902b-df34e4e1f34c">

Reply:
Updated.